### PR TITLE
added: decss.c <3

### PR DESCRIPTION
**Pull request by curtiszimmerman**

### CODE REVIEW

File: decss.c [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+  t4=t3&7;
+  t3=t3*2+8-t4;
+  sec+=0x80;
+  t5=0;
+  while(sec!=end) {
+  	t4=CSSt2[t2]^CSSt3[t1];
+  	t2=t1>>1;
+  	t1=((t1&1)<<8)^t4;
+  	t4=CSSt5[t4];
+  	t6=(((((((t3>>3)^t3)>>1)^t3)>>8)^t3)>>5)&0xff;
```
Comment (micahlmartin) [inline decss.c]: Extra `(`. Consider linting before committing.

File: decss.c [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+0xfc,0x7c,0xbc,0x3c,0xdc,0x5c,0x9c,0x1c,0xec,0x6c,0xac,0x2c,0xcc,0x4c,0x8c,0x0c,
+0xf4,0x74,0xb4,0x34,0xd4,0x54,0x94,0x14,0xe4,0x64,0xa4,0x24,0xc4,0x44,0x84,0x04,
+0xf8,0x78,0xb8,0x38,0xd8,0x58,0x98,0x18,0xe8,0x68,0xa8,0x28,0xc8,0x48,0x88,0x08,
+0xf0,0x70,0xb0,0x30,0xd0,0x50,0x90,0x10,0xe0,0x60,0xa0,0x20,0xc0,0x40,0x80,0x00
+};
+
+void CSSdescramble(unsigned char *sec,unsigned char *key) {
+  unsigned int t1,t2,t3,t4,t5,t6;
+  unsigned char *end=sec+0x800;
+  t1=key[0]^sec[0x54]|0x100;
```
Comment (micahlmartin) [inline decss.c]: Consider using more descriptive variable names.

File: decss.c (hunk 1 above)
Comment (curtiszimmerman) [inline decss.c]: Where? Take note, this source is exactly the original DeCSS.c source you can find all over the internet. ;)

Comment (curtiszimmerman) [inline decss.c]: [For example](http://decss.zoy.org/decss.c)...

File: decss.c (hunk 2 above)
Comment (curtiszimmerman) [inline decss.c]: This made me lol :)

**Pull request merged**